bin/brew

```diff
@@ -19,6 +19,13 @@ symlink_target_directory() {
   quiet_cd "$directory" && quiet_cd "$target_dirname" && pwd -P
 }
 
+for cmd in $(compgen -A builtin)
+do
+  unset -f $cmd  # Unset functions that override Bash builtins
```

Comment:
Could these comments be replaced with a single/block comment above the `for cmd`? Thanks!

Reply:
sure!